src/pynwb/form/build/map.py

```diff
@@ -327,10 +333,18 @@ def map_spec(self, **kwargs):
         self.map_const_arg(attr_carg, spec)
         self.map_attr(attr_carg, spec)
 
-    def __get_override_carg(self, name, builder, manager):
+    # def hack_get_override_carg(self, *args, **kwargs):
+    #     return self.__get_override_carg(*args, **kwargs)
+
+    def __get_override_carg(self, *args):
+        name = args[0]
+        remaining_args = tuple(args[1:])
         if name in self.constructor_args:
             func = self.constructor_args[name]
-            return func(self, builder, manager)
+            try:
+                return func(self, *remaining_args)
+            except TypeError:
+                return func(self, *remaining_args[:-1])
```

Comment:
Some overrides require manager as well.  This gets us past the need to change all of those function signatures for now, but should be cleaned up later.